tests/security/test_smtp_ssl.py

```diff
@@ -122,5 +125,5 @@ def test_smtp_ssl_verification_bad_cert(
 
 
 if __name__ == "__main__":
-    server = BadCertSMTPServer((SMTP_SERVER_HOST, SMTP_SERVER_PORT), (None, None))
+    server = BadCertSMTPServer((SMTP_SERVER_HOST, 0), (None, None))
```

Comment:
This was not even working earlier since SMTP_SERVER_PORT was not defined, now it binds to random port.